File: cpp/Program/Orang.cpp
#include <string>
using namespace std;

class Orang
{
protected:
    int id;
    string nik;
    string nama;

public:
    Orang()
    {
    }

    Orang(int id, string nik, string nama)
    {
        this->id = id;
        this->nik = nik;
        this->nama = nama;
    }

    // setter
    void setId(int id)
    {
        this->id = id;
    }

    void setNik(string nik)
    {
        this->nik = nik;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    // getter
    int getId()
    {
        return id;
    }

    string getNik()
    {
        return nik;
    }

    string getNama()
    {
        return nama;
    }
};

File: python/Program/Orang.py
class Orang:
    def __init__(self, id=None, nik="", nama=""):
        self._id = id
        self._nik = nik
        self._nama = nama

    # setter
    def set_id(self, id):
        self._id = id

    def set_nik(self, nik):
        self._nik = nik

    def set_nama(self, nama):
        self._nama = nama

    # getter
    def get_id(self):
        return self._id

    def get_nik(self):
        return self._nik

    def get_nama(self):
        return self._nama
